### PR TITLE
Unity Install 0.1.0

### PR DESCRIPTION
**Pull request by bdrelling**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by bdrelling** [comment]
Bitrise failed, but when I try to view details it 404's. Can't see what's up with it.

File: steps/unity-install/0.1.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  should_build_ios: true
+- opts:
+    description: |
+      Downloads and installs Unity Android Target Support packages.
+    is_expand: false
+    is_required: true
+    summary: Downloads and installs Unity Android Target Support packages.
+    title: Build Android?
+    value_options:
+    - true
```
Comment (lpusok) [inline steps/unity-install/0.1.0/step.yml]: Also should be a string.

File: steps/unity-install/0.1.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  git: https://bdrelling@github.com/bdrelling/bitrise-step-unity-install.git
+  commit: 63ce02bbaf1443921cd44f382cbf3feac9647875
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
+type_tags:
+- installer
+toolkit:
```
Comment (lpusok) [inline steps/unity-install/0.1.0/step.yml]: Can remove `toolkit` as step.sh is used by default if not specified.

File: steps/unity-install/0.1.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+  bash:
+    entry_file: step.sh
+deps:
```
[the remark this answers is not in the document]
Comment (bdrelling) [reply steps/unity-install/0.1.0/step.yml]: I left what was generated there when creating the step, assuming they were base requirements. Deleted this section, thanks.

File: steps/unity-install/0.1.0/step.yml (hunk 2 above)
Comment (bdrelling) [reply steps/unity-install/0.1.0/step.yml]: I assume then that `bash` is the default as well? Removing that section.

**Comment by bdrelling** [comment]
Closing per @lpusok.

**Pull request closed without merging**